object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
+      description: 'Tells if action bar button or link is disabled or ties it to selection',
+      oneOf: [
+        {
+          type: 'boolean',
+          default: false,
+        }, {
+          type: 'object',
+          additionalProperties: false,
+          description: 'Shortcut to enable and disable button or link as selection changes',
+          properties: {
+            minSelected: {type: 'number'},
+            maxSelected: {type: 'number'}
+          }
+        }
+      ]
+    },
+    actionItem: {
+      type: 'object',
+      properties: {
+        id: {
+          type: 'string',
+          description: 'Given to action handler to identify button or link'
+        },
+        text: {
+          type: 'string',
+          description: 'The text displayed on button or link'
+        },
+        size: {
+          type: 'string',
+          description: 'Size of button or link',
+          enum: ['small', 'medium', 'large'],
+          default: 'small'
```

Comment:
If UX changes their mind it's something I want to be applied globally (again, intrinsic to the object browser) rather than having some implementations that just forgot to update to the new spec - this really shouldn't be something a consumer can decide